kubernetes/create_cluster/script.md

```diff
@@ -75,7 +75,7 @@ credentials. We can specific them in the command if we want to (see
 minutes.
 
 ```
-az acs create --orchestrator-type=kubernetes --resource-group=$ACS_RESOURCE_GROUP --name=$ACS_CLUSTER_NAME --generate-ssh-keys
+az acs create --orchestrator-type=kubernetes --resource-group=$ACS_RESOURCE_GROUP --name=$ACS_CLUSTER_NAME --generate-ssh-keys --service-principal ${SERVICE_PRINCIPAL_ID} --client-secret ${SERVICE_PRINCIPAL_SECRET_KEY}
```

Comment:
I agree. The issue I'm working around currently is that service principals cannot currently mind other service principals. I'm working with the AD team to figure out if this is possible. In all our other tools we use a pre-created SP.